.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -0,0 +1,51 @@
+title: "[Q&A] "
+labels: ["q&a"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please fill out the form so the community can better assist you!
+  - type: input
+    attributes:
+      label: Python version (`python3 -V`)
+      description: Python version (`python3 -V`)
+      placeholder: "3.8"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: NVFlare version (`python3 -m pip list | grep "nvflare"`)
+      description: NVFlare version (`python3 -m pip list | grep "nvflare"`)
+      placeholder: "2.3.8"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)
+      description: NVFlare branch (if running examples, please use the branch that corresponds to the NVFlare version, `git branch`)
+      placeholder: "2.3"
+  - type: input
+    attributes:
+      label: Operating system
+      description: Operating system (Ubuntu, MacOS, Windows, etc)
+      placeholder: "Ubuntu 20.04"
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Have you successfully run the following examples?
+      options:
+        - label: hello-numpy-sag with simulator
```

Comment:
Do we need to ask for all of them? Maybe split it up into several yes/no questions for each of the examples.

Reply:
This is going to render as 4 check boxes that is not mandatory.

Do you prefer it to be ask four times yes/no checkboxes? (Not mandatory as well)

Reply:
Ah. 4 checkboxes as now is fine. Just rephrase it as "Have you successfully run any of the following examples?"

Reply:
Thanks will do!